pkg/apiserver/experiment/experiment.go

```diff
@@ -852,6 +854,7 @@ func (s *Service) getExperimentDetail(c *gin.Context) {
 // @Param namespace path string true "namespace"
 // @Param name path string true "name"
 // @Param kind path string true "kind" Enums(PodChaos, IoChaos, NetworkChaos, TimeChaos, KernelChaos, StressChaos)
+// @Param force path string true "force" Enums(true, false)
```

Comment:
Do we want to make this true by default?

Reply:
I think it is possible to set a default value, so would "false" be more reasonable?

Reply:
Yes, we are using non-force delete before, in the frontend as well. I think we should use the same. WDYT?